cpg_workflows/workflow.py

```diff
@@ -64,6 +91,9 @@ def path_walk(expected, collected: set) -> set:
     >>> path_walk({'a': Path('b'),'c': {'d': 'e'}, {'f': Path('g')}}, set())
     {Path('b'), Path('g')}
     """
+    if collected is None:
```

Comment:
If this can be none, should you adjust the type annotation on the `def path_walk` line above?